components/Profile/SettingsModal.jsx

```diff
@@ -184,13 +221,45 @@ const SettingsModal = (props) => {
             </TwinView>
 
             <SubTitle>Color Theme</SubTitle>
-            <ColorButton selected={selectedColor === 'aqua'} onPress={() => { updateColorTheme('aqua'); }}>
-              <Image source={require('../../assets/blue.png')} />
-            </ColorButton>
-            <ColorButton selected={selectedColor === 'default'} onPress={() => { updateColorTheme('default'); }}>
-              <Image source={require('../../assets/purple.png')} />
-            </ColorButton>
+            <TwinView>
+              <ColorButton selected={selectedColor === 'aqua'} onPress={() => { updateColorTheme('aqua'); }} style={{ width: 132, height: 32 }}>
+                <Image
+                  source={require('../../assets/blue.png')}
+                  style={{ width: 130, height: 30 }}
+                />
+              </ColorButton>
+              <ColorButton selected={selectedColor === 'default'} onPress={() => { updateColorTheme('default'); }} style={{ width: 132, height: 32 }}>
+                <Image
+                  source={require('../../assets/purple.png')}
+                  style={{ width: 130, height: 30 }}
+                />
+              </ColorButton>
```

Comment:
Can we use percentages on these button widths so this component looks good on different devices